languages/python/docs/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,49 @@
-# How to implement an Python concept exercise
+# How to implement a Python concept exercise
 
-TODO: describe how to implement a concept exercise for the Python track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes the steps required to implement a concept exercise for the Python track. As this document is generic, the following placeholders are used:
 
-[csharp-implementing]: ../../csharp/docs/implementing-a-concept-exercise.md
+- `<SLUG>`: the name of the exercise in snake_case (e.g. `anonymous_methods`).
+- `<NAME>`: the name of the exercise in snake_case (e.g. `anonymous_methods`).
+
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not). This information can be found in the exercise's GitHub issue.
+
+To implement a concept exercise, the following files need to be created:
+
+```
+languages
+└── python
+    └── exercises
+        └── concept
+            └── <SLUG>
+                ├── .docs
+                |   ├── instructions.md
+                |   ├── introduction.md
+                |   ├── hints.md
+                |   └── after.md (optional)
+                ├── .meta
+                |   ├── config.json
+                |   └── <SLUG>.py
```

Comment:
We're using the `<SLUG>` placeholder in 3 different places. Are they all the same thing?

Reply:
See my comment [here](https://github.com/exercism/v3/pull/223/files#r368954390) for an updated tree ... `<SLUG>` should only appear in directory names.